src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -383,14 +381,19 @@ export default {
 
     loadCorrection (filing, section) {
       if (section) {
+        const localDateTime = this.convertUTCTimeToLocalTime(filing.header.date)
+
         const item = {
           type: filing.header.name,
+          title: `Correction - ${this.typeToTitle(filing.correction.correctedFilingType)}`,
+          filingId: filing.header.filingId,
           filingAuthor: filing.header.certifiedBy,
+          filingDateTime: localDateTime,
           filingDate: filing.correction.correctedFilingDate,
-          filingId: filing.header.filingId,
+          isCorrection: true,
```

Comment:
This ones new, where do we use this property?

Reply:
Appears it is for future use in `correctThisItem()` ?

Reply:
Yes, for future use.